Bukkit/build.gradle.kts

```
@@ -24,7 +24,7 @@ dependencies {
     implementation("org.bstats:bstats-bukkit")
 
     // Paper
-    compileOnly("io.papermc.paper:paper-api")
+    compileOnly("dev.folia:folia-api:1.19.4-R0.1-SNAPSHOT")
```

```suggestion
    compileOnly("dev.folia:folia-api:1.19.4-R0.1-SNAPSHOT")
```
This doesn't look right to me. The bukkit module has to stick to Paper for compatibility and compiling reasons.

---

When i allow both. The the API support for folia will break in IJ

---

But this breaks compatibility with Spigot and Paper.

---

Okay. Let me check that how we can handle it